procstats/proc_linux.go

```diff
@@ -1,25 +1,70 @@
 package procstats
 
 import (
+	"io/ioutil"
 	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"sync"
 	"syscall"
 	"time"
 
 	"github.com/segmentio/stats/procstats/linux"
 )
 
-func collectProcInfo(pid int) (info ProcInfo, err error) {
-	defer func() { err = convertPanicToError(recover()) }()
+var (
+	clockTickOnce  sync.Once
+	clockTickHertz uint64
+	clockTickError error
+)
 
-	pagesize := uint64(syscall.Getpagesize())
-	rusage := syscall.Rusage{}
+func clockTicksToDuration(ticks uint64) time.Duration {
+	clockTickOnce.Do(func() { clockTickHertz, clockTickError = clockTick() })
+	check(clockTickError)
+	return time.Duration(1e9 * float64(ticks) / float64(clockTickHertz))
```

Comment:
Why not use `time.Second` instead of `1e9`?

Reply:
`time.Second` is a `time.Duration`, but the expression here works with floats. It would have to be `float64(time.Second)` which is much more verbose.

I can change it if you feel strongly about it.

Reply:
I don't feel that strongly :)